ansible/roles/test/tasks/crm/crm_test_fdb_entry.yml

```diff
@@ -13,14 +13,16 @@
         crm_stats_fdb_entry_used: "{{ out.stdout_lines[0] }}"
         crm_stats_fdb_entry_available: "{{ out.stdout_lines[1] }}"
 
-    - name: Copy FDB JSON config to switch.
-      copy: src=roles/test/tasks/crm/fdb.json dest=/tmp
+    - set_fact: op="SET"
+
+    - name: Render FDB JSON config.
+      template: src=roles/test/templates/fdb_conf.j2 dest=/tmp/fdb_conf.json
```

Comment:
you rename the file to fdb_json.j2 but here you use fdb_conf.j2, obviously, there is a mismatch of the name. Can you provide your test run results? We would expect you have run the test successfully before you submit the fix.